proc_stat_test.go

```diff
@@ -76,16 +77,30 @@ func TestProcStat(t *testing.T) {
 	}
 }
 
-func TestProcStatIgnored(t *testing.T) {
+func TestProcStatLimits(t *testing.T) {
 	p, err := getProcFixtures(t).Proc(26232)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	_, err = p.Stat()
+	s, err := p.Stat()
 	if err != nil {
 		t.Errorf("want not error, have %s", err)
 	}
+
+	// max values of stat int fields
+	for _, test := range []struct {
+		name string
+		want int
+		have int
+	}{
+		{name: "waited for children user time", want: math.MinInt64, have: s.CUTime},
+		{name: "waited for children system time", want: math.MaxInt64, have: s.CSTime},
```

Comment:
Using MinInt64 / MaxInt64 here causes tests to fail on 32-bit archs, since these are untyped constants, being assigned to machine-native ints (in struct a few lines up).

Either the struct needs to explicitly use int64 types, or you need to use smaller constants.

(incidentally, this is not the first time that there has been a test regression for 32-bit arch in this package)

Reply:
Makes sense, do you want to open an issue for testing on 32 bit archs? We need to see how to make that possible (let alone who has time to work on it) but agreed that we should track that at least.

Reply:
I was able to reproduce the 32-bit test failure simply by exporting `GOARCH=386` before running the tests.